### ISSUE TITLE
Tallow not starting

### ISSUE THREAD

**Issue by MoNTE48** [basil00/TorWall#15]
Hello. My config - 
Windows 10 x64, build 1803
I try run portatable or install version, but after i run Tallow.exe and give admin right, nothing happens. I've tried it many times, rebooting is also not helping

**Comment by basil00** [comment]
First try running Tallow from the command line (as Administrator in the `c:\Program Files (x86)\Tallow` directory).  It should print some logging information which may indicate the problem.

**Comment by basil00** [comment]
It occurred to me that the problem may be related to [a recent WinDivert issue](https://github.com/basil00/Divert/issues/134).  I will release an updated version of Tallow in a day or two, and maybe that will resolve the problem.

**Comment by basil00** [comment]
A new version of Tallow is now available: https://github.com/basil00/TorWall/releases/tag/v1.2

Not sure if this will fix the issue but it is worth trying.

**Comment by MoNTE48** [comment]
I try run Tallow  in command line
```
C:\WINDOWS\system32>cd C:\Program Files (x86)\Tallow

C:\Program Files (x86)\Tallow>tallow.exe

C:\Program Files (x86)\Tallow>tallow.exe
```
I try 1.2 ver - same problem... After i give admin permission - nothing happens

**Comment by basil00** [comment]
OK so Tallow literally does nothing, not even any output to the console.  This  is strange.

Are you running the console as Administrator?  Otherwise I'd guess that something is blocking Tallow from running, e.g., anti-virus/firewall.

**Comment by MoNTE48** [comment]
@basil00 yes, as admin. I now use any anti-virus or firewall. Only default Windows 10 Defender.

**Comment by basil00** [comment]
I am not sure what is causing the issue, unless Defender is somehow blocking it.

**Comment by MoNTE48** [comment]
Does not look like. Otherwise, the defender would issue a notification.Can you make a debug build with a full log output to the console?

**Comment by basil00** [comment]
I will create a version with more logging, but will have to get back on this.

**Comment by basil00** [comment]
I added more [debugging output during initialization](https://github.com/basil00/TorWall/commit/63c8f2dbcd7c97c9491e45d853f62a53b9817897).  The version 1.2 binaries have been updated and can be downloaded from here: https://github.com/basil00/TorWall/releases/tag/v1.2

First, uninstall (or delete) the old version, then try running the new version from the console.  It should print a copyright message at the very least.
